src/Stratis.Bitcoin.Features.PoA/Voting/PollResultExecutor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.PoA.Voting
+{
+    public interface IPollResultExecutor
```

Comment:
The policy requires at least a high level description of this interface and methods

Reply:
Ok, will comment it in the next PR